sweep/sweeper.go

```diff
@@ -1041,36 +1023,6 @@ func (s *UtxoSweeper) markInputsPendingPublish(tr *TxRecord,
 
 		// Record another publish attempt.
 		pi.publishAttempts++
-
```

Comment:
Can we also get rid of `publishAttempts` here? Elsewhere, we used this to detect if an input was published or not, but now we have `SweepState`. We could even morph things slightly and store the amount of attempts directly into the state.